src/main/java/seedu/address/commons/core/Config.java

```diff
@@ -32,6 +33,14 @@ public void setUserPrefsFilePath(Path userPrefsFilePath) {
         this.userPrefsFilePath = userPrefsFilePath;
     }
 
+    public boolean getIsTestEnv() {
+        return testEnv;
+    }
+
+    public void setTestEnv (boolean testEnv) {
+        this.testEnv = testEnv;
+    }
+
```

Comment:
is not being used through the codes, can refer to setuserPrefsFilePath and see how it is being used.
Maybe can insert a comment saying what it is for or how to use it

Reply:
Deleted the method